routes/api/crowbar/repocheck.js

```diff
@@ -1,14 +1,35 @@
 var express = require('express'),
-    router = express.Router();
+    router = express.Router(),
+    repo_correct = false;
 
 /* GET Admin Repo Checks. */
 router.get('/', function(req, res) {
-    res.status(200).json({
-        'SLES_12_SP2': true,
-        'SLES_12_SP2_Updates': true,
-        'SLES_OpenStack_Cloud_7': true,
-        'SLES_OpenStack_Cloud_7_Updates': true
-    });
+    var data = {
+        'os': {
+            'available': true,
+            'repos': {}
+        },
+        'openstack': {
+            'available': false,
+            'repos': {
+                'x86_64': {
+                    'missing': [
+                        'SUSE OpenStack Cloud 7',
```

Comment:
That's what you should've changed as part of https://trello.com/c/sl8vHEaY/146-3-s49p14-bsc-999946-upgrade-check-for-repositories-on-admin-server-only-checks-for-product